web/viewer.html

```diff
@@ -300,7 +300,7 @@
                         </div>
                         <div class="editorParamsSetter">
                           <label for="editorInkOpacity" class="editorParamsLabel" data-l10n-id="pdfjs-editor-ink-opacity-input">Opacity</label>
-                          <input type="range" id="editorInkOpacity" class="editorParamsSlider" value="100" min="1" max="100" step="1" tabindex="0">
+                          <input type="range" id="editorInkOpacity" class="editorParamsSlider" value="1" min="0.1" max="1" step="0.1" tabindex="0">
```

Comment:
This gives a lot less control over the opacity, from 100 to 10 possible values, was than intended?

(For comparison, the `editorInkThickness` has 20 possible values.)